### ISSUE TITLE
File name not working

### ISSUE THREAD

**Issue by Sawalhah** [glebovpavel/IR_to_MSExcel#89]
Hi,
I'm using apex version 20.1 with IG and it works perfect,
I've added a custom file name in File Name property as in documentation
![image](https://user-images.githubusercontent.com/10211157/95243252-079f0a00-0819-11eb-8e0e-8ebcac012991.png)

but when I download the report, it takes the default file name "Excel.xlsx"

any help please

**Comment by Astrid0678** [comment]
Hi,

Following..
I have the same issue.   It only seems to work for interactive Reports.

Thx!

**Comment by bogdanbilcan** [comment]
Hi,

Same version of Apex here. 20.1.
The excel file name is changed to what I put in that field there but it does not read any substitution value for IR.
I tried to use the example in documentation with Export-&SYSDATE. and the resulted name was only Export-.xlsx
I tried to create a hidden or visible or display only item on page with some static value in it, and put that item in Filename field with &P1_NAME. and filename was empty on download.

Thx.
